packages/server/package.json

```diff
@@ -15,7 +15,7 @@
     "test": "mocha ./test/setup test/**/*.test.ts",
     "mocha": "mocha ./test/setup",
     "test:app:start": "ts-node ./bin/todomvc.ts",
-    "generate:schema": "BIGTEST_GENERATE_SCHEMA=true ts-node ./src/schema.ts",
+    "generate:schema": "env BIGTEST_GENERATE_SCHEMA=true ts-node ./src/schema.ts",
```

Comment:
Prepended `env` in a node script is afaik the most cross compatible way to inline an env var.